CHANGELOG.rst

```diff
@@ -25,6 +25,10 @@ Changed
 
   Contributed by @hnanchahal
 
+* Updated Travis CI and CircleCI builds to only use Python 3
```

Comment:
Please don't forget to include the PR number #5090 also the type of the change.

Also is it (improvement) or (cleanup)? Maybe `Removed` section below would fit this better, considering it's breaking change nature.